openpower/configs/witherspoon_defconfig

```diff
@@ -59,6 +59,8 @@ BR2_PACKAGE_PETITBOOT=y
 BR2_PACKAGE_PETITBOOT_MTD=y
 BR2_OCC_BIN_FILENAME="occ.bin"
 BR2_OCC_GPU_BIN_BUILD=y
+BR2_HCODE_INCLUDE_IONV=y
+BR2_USE_PRE_COMPILED_HCODE=y
```

Comment:
Why not build from source? We don't have any other components shipping prebuilt binaries when we have source to them.